src/server/api/controllers/topics.controller.js

```diff
@@ -12,7 +12,7 @@ const getTopics = async () => {
 const getTopicById = async (id) => {
   try {
     const topics = await knex('topics')
-      .select('topics.id as id', 'title')
+      .select('topics.id as id', 'topic_name')
       .where({ id });
     if (topics.length === 0) {
```

Comment:
In line nr 17, typo error? , in this  if(topics.length===0) statement "topics" should be "topic" instead.

Reply:
You always get a collection back - and then you check how many there are inside.